src/pages/Models/Details/generators.js

```diff
@@ -99,10 +108,13 @@ export function generateApplicationRows(
           content: app.status ? generateStatusElement(app.status.status) : "-",
           className: "u-capitalise",
         },
-        { content: "-", className: "u-align--right" },
-        { content: "-", className: "u-align--right" },
+        { content: app.workloadVersion || "-", className: "u-align--right" },
+        { content: Object.keys(app.units).length, className: "u-align--right" },
```

Comment:
What is `app.units` is not available or not an array?

Reply:
The application data set always has an empty array for the units as part of the contract but I suppose it's possible that could change. I've made it more resilient.